src/elements/list-box.svelte

```diff
@@ -0,0 +1,159 @@
+<svelte:options immutable tag='v-list-box' />
+
+<script lang='ts'>
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+// disabled to pass untyped object to cx library in template
+
+import cx from 'classnames';
+import { afterUpdate } from 'svelte';
+
+import { dispatcher } from '../lib/dispatch';
+import { addStyles } from '../lib/index';
+import { htmlToBoolean } from '../lib/boolean';
+
+export let disabled: string;
+export let left = '';
+export let right = ''; 
+export let leftlabel = ''; 
+export let rightlabel = ''; 
+export let height = '200px';
+
```

Comment:
Would add empty state value options for both the left and right, so when there are no options, then the user can input "No models on robot" or any other info. Ideally make sure that it is linkable so people can get to the documentation or anything else that is necessary

Reply:
Oh you right I actually completely forgot about this. Will add